### ISSUE TITLE
Any comment about upcoming Git Rev News edition 29

### ISSUE THREAD

**Issue by chriscool** [git/git.github.io#251]
A currently mostly empty draft is there:

https://github.com/git/git.github.io/blob/master/rev_news/drafts/edition-29.md

Feel free to comment in this issue, or to use the edit button (that looks like a pen) to edit and create a pull request with the changes you would like.

Thanks!

cc @tfnico @jnareb @mjaix @gitster

**Comment by jnareb** [comment]
I don't know where to put [Simul - Version Control for Microsoft Word: Bringing the features of Git and GitHub to Microsoft Word](https://www.simuldocs.com/blog/ben-morris/version-control-for-microsoft-word): **Various** or **Light reading**; I think **Git tools and sites** would not be a good fit...

**Comment by tfnico** [comment]
@jnareb I don't think it matters much. Just put it where it looks nice :)

**Comment by jnareb** [comment]
added (together with a few other links)

**Comment by jnareb** [comment]
Is Git Rev News edition 29 planned to be released on this Wednesday (July 12th), or next week?

In any case it would be good to think about sending interview questions early.

**Comment by jnareb** [comment]
and a few links more in #254

**Comment by chriscool** [comment]
@jnareb It will be next week. I agree about doing things early but right now there are a lot of other things I have to do before I can take care of that. Also I am often trying to interview someone who was involved in the articles and it is difficult to do that before finding which article I will write.
But if you think about or can find someone to interview early, you are very welcome to do it even if the articles will not be related to that person.
Thanks for the links!

**Comment by jnareb** [comment]
I guess this means that the [Emacs and Magit](https://lwn.net/Articles/727550/) article on LWN.net is currently subscriber-only, and it will be free only after a week ([subscriber lin](https://lwn.net/SubscriberLink/727550/cdbd03d32f461e4a/)). I guess this means that it would be better postponed to the next release (I'd prefer to not abuse the "subscriber link" feature).

**Comment by chriscool** [comment]
@jnareb yeah let's wait for the next edition.

**Comment by chriscool** [comment]
I just sent the draft for this edition. Sorry I had time for only one article (though a quite big one).

**Comment by mjaix** [comment]
Hi,
start trying to fix typos after the usual work @ home ... 
Cheers,
Markus

**Comment by tfnico** [comment]
Just pushed my links and releases. Please pull before editing more, @mjaix :)

**Comment by mjaix** [comment]
Finished my work, too - feel free to change.
Good article about really good work.
No GitHub traffic jam this time :-).
Cheers,
Markus
a { cursor: text; }

**Comment by chriscool** [comment]
Ok, I improved the formating of some code in a comment and published. Thanks everyone!
